components/dashboard/script.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+
+const vars = {
+    REACT_APP_HOSTNAME : "Gitpod",
+    REACT_APP_DOCS_URL : "https://www.gitpod.io/docs/",
+    REACT_APP_SOCIAL_URL : "https://community.gitpod.io/",
+    REACT_APP_FAVICON_192 : "icons/favicon192.png",
+    REACT_APP_FAVICON_256 : "icons/favicon256.png",
+    REACT_APP_FAVICON_512 : "icons/favicon512.png",
+    REACT_APP_BRAND_SVG : "icons/gitpod.svg",
+    REACT_APP_LOGO : "icons/gitpod32.svg"
+};
+
+let data = "";
+for(const prop in vars) {
+    data += prop + "=" + vars[prop] + "\n";
+}
+fs.writeFileSync('.env', data);
```

Comment:
<details>
<summary>:warning: <strong>This code is missing a blank line after the code block and before the next statement or before the comment.</strong> (click for details)</summary>

Consider leaving a single blank line after blocks/group of statements and before the next statement or before the comment unless it's on the first line of a block.  
  
Adding a blank line improves code readability and keeps the style consistent.

**[More details](https://confluence.devfactory.com/x/hnvlFQ)**

## The improvement suggestion in this case
Consider adding a single blank line before the statement or comment.  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="51f1d1ed-0bb5-4d49-b3d8-f18988a4d9fe" data-finding-id="367360902" href="none" codereview-uuid-reserved-tag data-business-id="3d94a8d2-7431-4dbe-bce7-80b87bb68d39" data-analyzer="js" />)